tensorflow/python/lib/core/ndarray_tensor.cc

```diff
@@ -216,7 +216,13 @@ Status PyArray_TYPE_to_TF_DataType(PyArrayObject* array,
       } else if (pyarray_type == custom_dtypes.float8_e4m3fn) {
         *out_tf_datatype = TF_FLOAT8_E4M3FN;
         break;
-      } else if (pyarray_type == custom_dtypes.int4) {
+      } else if (pyarray_type == custom_dtypes.float8_e5m2fnuz) {
+        *out_tf_datatype = TF_FLOAT8_E5M2FNUZ;
+        break;
+      } else if (pyarray_type == custom_dtypes.float8_e4m3fnuz) {
+        *out_tf_datatype = TF_FLOAT8_E4M3FNUZ;
+        break;
+      }else if (pyarray_type == custom_dtypes.int4) {
```

Comment:
nit: formatting for final else if.  Can you run the formatter?

Reply:
Hi @cantonios sorry for the typo, I have formatted it.